### PR TITLE
Add an integration test case for VideoDataset

### PR DESCRIPTION
**Pull request by yongtang**
In our build test we haven't had a test case yet
(only tested that `pip install tensorflow_io-*-*-*.whl` works),
this PR add an integration test case for VideoDataset.

This PR is part of #57.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

File: tests/test_video.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+import tensorflow as tf
+from tensorflow.keras.applications.resnet50 import ResNet50
+from tensorflow.keras.applications.resnet50 import preprocess_input, decode_predictions
+from tensorflow_io.video import VideoDataset
+
+video_path = os.path.abspath(
+    os.path.join(os.path.dirname(os.path.abspath(__file__)),
+    "..",
+    "tensorflow_io/video/python/kernel_tests/testdata/small.mp4"))
```
Comment (terrytangyuan) [inline tests/test_video.py]: Should we move all test data to top level since it's used by top level `tests/` as well?

File: tests/test_video.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+  dataset = VideoDataset([video_path]).repeat(num_repeats)
+  iterator = dataset.make_initializable_iterator()
+  init_op = iterator.initializer
+  get_next = iterator.get_next()
+
+  with tf.Session() as sess:
+    sess.run(init_op)
+    for _ in range(num_repeats):  # Dataset is repeated.
+      for _ in range(166): # 166 frames
```
Comment (terrytangyuan) [inline tests/test_video.py]: You can remove the comment since the code is self-explanatory.

Comment (yongtang) [reply tests/test_video.py]: Updated. Thanks.

File: tests/test_video.py (hunk 1 above)
Comment (yongtang) [reply tests/test_video.py]: The only issue is the bazel as `testdata` directory needs to be visible to the project e.g., `//tensorflow_io/video/BUILD`, haven't found out a successful way to make it work with bazel yet.

I can continue investigation and add a follow up PR once I am able to expose the testdata in bazel.

**Pull request merged**